src/Microsoft.Extensions.CodeGeneration.Utils/DotNet/LibraryExporterExtensions.cs

```diff
@@ -21,29 +21,31 @@ public static string GetResolvedPathForDependency(this ILibraryExporter _library
             {
                 throw new ArgumentNullException(nameof(library));
             }
-
             var exports = _libraryExporter.GetAllExports();
+            if(exports == null)
```

Comment:
Well, ideally they shouldn't be. But I did run into Null reference exceptions on net451 process while converting the DefaultAssemblyLoadContext, so kept it here as a check.